src/plugins/dashboard/public/dashboard_actions/filters_notification_action.tsx

```diff
@@ -76,8 +76,6 @@ export class FiltersNotificationAction implements Action<FiltersNotificationActi
     );
   };
 
-  public readonly MenuItem = reactToUiComponent(this.FilterIconButton);
```

Comment:
Now that reactToUiComponent no longer loads this component in a separate react tree, can you remove `I18nProvider` and `KibanaThemeProvider` wrappers in `MenuItem`?

Reply:
That is the goal&mdash;to be able to preserve the React context, in general. But in this specific case, I don't know, it needs to be tested case-by-case, as React context might have been broken in some other place as well.